### PR TITLE
Refactor duplicated function expression for greeting

### PR DESCRIPTION
**Pull request by kencheung**

### CODE REVIEW

**Comment by benjamn** [comment]
This looks good to me, but I have to say this would be a lovely application of ECMAScript 2015 [template strings](http://www.2ality.com/2015/01/template-strings-html.html).

Would you be willing to add `api.use("ecmascript")` to the `Package.onUse` callback in `meteor/packages/accounts-password/package.js`, and then use a multi-line template string?

Something like this:
```js
      return `${greeting}

${welcomeMsg}, simply...
Thanks.
`;
```

**Comment by kencheung** [comment]
I have made the changes as suggested. Please advice if I misunderstand anything. Thank you.

File: packages/accounts-password/email_templates.js [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+
+function greet(welcomeMsg) {
+  return function(user, url) {
+      var greeting = (user.profile && user.profile.name) ?
+            ("Hello " + user.profile.name + ",") : "Hello,";
+      return `${greeting}
+
+${welcomeMsg}, simply click the link below.
+
+url
```
Comment (benjamn) [inline packages/accounts-password/email_templates.js]: `${url}`?

Comment (kencheung) [inline packages/accounts-password/email_templates.js]: Fixed. Thank you.

**Comment by benjamn** [comment]
Perfect, thanks!

**Pull request merged**